spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/JpaAutoConfiguration.java

```diff
@@ -46,7 +46,8 @@
 @ConditionalOnBean(EntityManagerFactory.class)
 @RegisterDefaultEntities(packages = {
         "org.axonframework.eventhandling.tokenstore",
-        "org.axonframework.modelling.saga.repository.jpa"
```

Comment:
I might not entirely follow what this change is for. Can you enlighten me?

Reply:
When the EntityManager is created, it is told by spring what entities should be used. This annotation scans the paths provided and provides the found classes to the EntityManager/hibernate for usage. Without it, the JPA dead letter entity would never be found when using Spring, and would not work at all.